test/functional/services/common/index.ts

```diff
@@ -9,7 +9,8 @@
 export type { Browser } from './browser';
 export { BrowserProvider } from './browser';
 export { FailureDebuggingProvider } from './failure_debugging';
-export { FindProvider } from './find';
+// TODO-TRE: I get the feeling this should not be exported here.
+// export { FindProvider } from '@kbn/ftr-common-functional-ui-services';
```

Comment:
Yes, this is no longer needed here as it's imported from the package now in `test/functional/services/index.ts`.